homeassistant/components/escea/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""Platform for the Escea fireplace."""
+
+from homeassistant import config_entries
+from homeassistant.components.climate import DOMAIN as CLIMATE_DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .discovery import async_start_discovery_service, async_stop_discovery_service
+
+PLATFORMS = [CLIMATE_DOMAIN]
+
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: config_entries.ConfigEntry):
```

Comment:
Missing return value typing. Please type the whole signature when adding type annotations.

Reply:
Thanks - fixed throughout